catalog/users/zeroboard.json

```diff
@@ -0,0 +1,7 @@
+{
+  "id": "zeroboard",                                                         
+  "kind": "solutionprovider",                                
+  "name": "Zeroboard Inc.",                    
+  "email": "riku.isayama@zeroboard.jp",
+  "website": "https://zeroboard.jp/"
```

Comment:
While the `user` `json` is valid as is, you might want to add the optional field
```json
"logo": "<url-to-your-logo>"
```
That ensures that in your logo is displayed in the collaborators page.

I am sorry that the possibility of adding the `"logo"` field was not explicit in the documentation. I have updated it meanwhile.

Reply:
@raimundo-henriques Thank you for your comment and the advice!
I've added our logo to zeroboard.json, so it would be great if you could review it once more. Thank you!